plan/planbuilder.go

```diff
@@ -1030,61 +1030,72 @@ func (b *planBuilder) buildInsert(insert *ast.InsertStmt) Plan {
 		})
 	}
 
+	insertPlan.Schema4OnDuplicate = insertPlan.tableSchema
+	if insert.Select != nil {
+		selectPlan := b.build(insert.Select)
+		if b.err != nil {
+			return nil
+		}
+
+		// If the schema of selectPlan contains any generated column, raises error.
```

Comment:
It is misleading that it checks the schema of the insert table, not the select plan.